test/javascripts/src/format24_test.coffee

```diff
@@ -44,3 +44,58 @@ testGroup "format24 (24h time)", ->
       assert.equal getText(el), now.format("h:mma")
       config.useFormat24 = originalFormat24
       done()
+
+  testAsync "turned off relative time", (done) ->
+    ago = moment().subtract("days", 5)
+    day  = ago.format "dddd"
+    time = ago.format "h:mma"
+    expected = "#{day} at #{time}"
+
+    originalFormat24 = config.useFormat24
+    config.useFormat24 = false
+
+    el = addTimeEl type: "time-ago", datetime: ago.toISOString()
+    defer ->
+      assert.equal getText(el), expected
+      config.useFormat24 = originalFormat24
+      done()
+
+  testAsync "turned on relative time", (done) ->
+    ago = moment().subtract("days", 5)
+    day  = ago.format "dddd"
+    time = ago.format "H:mm"
+    expected = "#{day} at #{time}"
+
+    originalFormat24 = config.useFormat24
+    config.useFormat24 = true
+
+    el = addTimeEl type: "time-ago", datetime: ago.toISOString()
+    defer ->
+      assert.equal getText(el), expected
+      config.useFormat24 = originalFormat24
+      done()
+
+  testAsync "turned off title", (done) ->
+    ago = moment().subtract("days", 5)
+
+    originalFormat24 = config.useFormat24
+    config.useFormat24 = false
+
+    el = addTimeEl type: "time-ago", datetime: ago.toISOString()
+    defer ->
+      assert.equal getTitle(el), ago.format('MMMM D, YYYY [at] h:mma ');
+      config.useFormat24 = originalFormat24
+      done()
```

Comment:
```suggestion
  testAsync "element title when turned off", (done) ->
    ago = moment().subtract("days", 5)
    originalFormat24 = config.useFormat24
    config.useFormat24 = false

    el = addTimeEl type: "time-ago", datetime: ago.toISOString()
    defer ->
      assert.equal getTitle(el), ago.format("MMMM D, YYYY [at] h:mma");
      config.useFormat24 = originalFormat24
      done()
```

The other tests don't have that empty line between the subject variable assignment and the config. It's not wrong or bad, just making the change for consistency. Same goes for using double quotes instead of single ones.

Why is the space at the end of the expected text necessary? Does the result of `ago.format('MMMM D, YYYY [at] h:mma ')` include that space? If so, ideally we'd avoid that, assuming it's not an existing bug in the library.

Let's make the same edits to the `turned on title` test.